packages/loader/container-loader/src/test/serializedStateManager.spec.ts

```diff
@@ -5,7 +5,7 @@
 
 import { strict as assert } from "assert";
 
-import { stringToBuffer } from "@fluid-internal/client-utils";
```

Comment:
I did not look through very carefully through test coverage.
Do we have fuzz test? If not, I think we should invest into it. UTs and E2E tests are required, but our mental ability to foresee all the interesting combinations here are limited. Service tests will for sure eventually catch issues, but it's a bit too late - very hard to even diagnose the problem.